src/app/QuickStarts/quickstarts/start-a-recording.tsx

```diff
@@ -80,18 +78,16 @@ In this quick start, you will use [APP] to connect to a target JVM and start a r
     tasks: [
       {
         title: 'Go to the Recordings tab',
-        description:
-          '1. Click the [Recordings]{{highlight nav-recordings-tab}} tab in the [APP] console navigation bar.',
+        description: '1. In the [APP] console navigation bar, click [Recordings]{{highlight nav-recordings-tab}}.',
         review: {
           instructions: '#### Verify that you see the Recordings page.',
           failedTaskHelp:
-            'If you do not see the navigation bar, you can click the `☰` button in the [top left corner of the page]{{highlight nav-toggle-btn}}.',
+            'If you do not see the navigation bar, click the [menu button]{{highlight nav-toggle-btn}} on the masthead.',
         },
       },
       {
         title: 'Select a target JVM',
         description: `    
```

Comment:
I cant comment on that line so Just wanna call out here:

Line 100: ```failedTaskHelp: 'If you do not see the table, try the steps again.',```

The users might wonder `What steps?`. Lets do `try the steps above again.`